composer.json

```diff
@@ -24,7 +24,7 @@
         "symfony/framework-bundle": "^3.4 | ^4.4 | ^5.0 | ^6.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^9.5",
+        "phpunit/phpunit": "^9.6 | ^10.1",
```

Comment:
it doesn't make much sense to keep two versions for a dev tool like PHPUnit. Let's drop version 9

Reply:
PHPUnit 10 requires PHP 8.1. As long as we need to run tests on PHP 7, we need PHPUnit 9. And dropping support for PHP 7 is probably out of scope for this PR?

Reply:
good point